app/components/primer/avatar_component.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Primer
+  class AvatarComponent < Primer::Component
+    SMALL_THRESHOLD = 24
+
+    def initialize(src:, alt:, size: 20, square: false, **kwargs)
+      @kwargs = kwargs
+      @kwargs[:tag] = :img
+      @kwargs[:src] = src
+      @kwargs[:alt] = alt
+      @kwargs[:size] = size
+      @kwargs[:height] = size
+      @kwargs[:width] = size
+
+      @kwargs[:classes] = class_names(
+        "avatar",
+        kwargs[:classes],
```

Comment:
It's a bit of a nitpick, but do you think we should have a standard practice for where we inject `kwargs[:classes]` in this kind of pattern? I could see it making sense to always have them be the last argument to `class_names`, or maybe the first?

Reply:
adding it to the end will cause an error because the other argumets are `kwargs`

Reply:
Ah, so maybe we standardize on having it first?

Reply:
I do prefer having the component main class first 🤷

Reply:
I'm not certain, but doesn't the order of the classes matter in terms of how CSS applies them? That's the reason I was thinking it would be nice to have the custom classes first or last, vs. in the middle.

Reply:
it shouldn't matter
what really matters is the order they are defined in the CSS file

Reply:
![image](https://user-images.githubusercontent.com/11280312/90149365-0803b300-dd4a-11ea-8714-629e6427f03e.png)

`ml-2` is defined after `ml-1` in the file, so it is more `important`